VERSION

```diff
@@ -1 +1 @@
-v0.25.0
\ No newline at end of file
+v0.26.0-dev
```

Comment:
This is already done by https://github.com/gardener/logging/commit/48e2dba19a34d44d12f16e094516a5b82c2dac89